src/Raven.Server/Web/System/OngoingTasksHandler.cs

```diff
@@ -487,7 +487,8 @@ public async Task BackupDatabaseOnce()
                                     {
                                         var runningBackupStatus = new PeriodicBackupStatus { TaskId = 0, BackupType = backupConfiguration.BackupType };
                                         var backupResult = backupTask.RunPeriodicBackup(onProgress, ref runningBackupStatus);
-                                        BackupTask.SaveBackupStatus(runningBackupStatus, Database, Logger, backupResult);
+                                        AsyncHelpers.RunSync(() => 
```

Comment:
We can `await` here.

Reply:
Done.

Reply:
`PoolOfThreads.GlobalRavenThreadPool.LongRunning` cannot be async, so returned it back to `RunSync`